rust-toolchain.toml

```diff
@@ -3,5 +3,5 @@
 # and update the CI base image version everywhere
 channel = "1.84"
 components = [ "clippy", "rust-docs", "rustfmt","rustc-dev", "rustc", "rust-src"]
-targets = [ "wasm32-unknown-unknown" ]
+targets = [ "wasm32v1-none" ]
```

Comment:
Do we need to update all the places that previously used the old target?
- https://github.com/frequency-chain/frequency/blob/5ea8307f60a1d2c154dec304615c9a9086d43ad9/.github/workflows/verify-pr-commit.yml#L206
- Readme
- deny.toml

Reply:
Done